components/wizard/WizardAdminAccount.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <v-form ref="form" v-model="validInputs">
+    <v-text-field
+      v-model="admin.Name"
+      outlined
+      :label="$t('username')"
+      type="username"
+      required
+    ></v-text-field>
+    <v-text-field
+      v-model="admin.Password"
+      outlined
+      :label="$t('password')"
+      :rules="[(v) => !!v || $t('passwordRequired')]"
+      required
```

Comment:
Passwords aren't required here in the current web client.

Reply:
Imho we should make the admin account have a password. Regardless, I have removed it.